assets/js/appVue.js

```diff
@@ -0,0 +1,171 @@
+        ],
+        "contentType": "multipart/alternative; boundary=0000000000009b2af005950b22c2",
+        "date": "2019-10-16T18:18:21.000+0000",
+        "from": [
+            "Max Zvyagin <deathstingray@gmail.com>"
+        ],
+        "recipients": [
+            "ikpi63_holding@rambler.ru"
+        ],
+        "folder": "INBOX"
+    },
+    {
+        "numberInFolder": 5,
+        "title": "HI GUYS AND GIRLS",
+        "content": [
+            "hi world\n\n\n-- \nВалентин Бодров",
+            "\n<HTML><BODY>hi world<br><br><br>-- <br>Валентин Бодров</BODY></HTML>\n"
+        ],
+        "contentType": "multipart/alternative; \r\n\tboundary=--ALT--AFNoi4Q90b3QTjzpwrWtAEFYGmxz37BT1571250156",
+        "date": "2019-10-16T18:22:36.000+0000",
+        "from": [
+            "\"Валентин Бодров\" <valbod@mail.ru>"
+        ],
+        "recipients": [
+            "ikpi63_holding@rambler.ru"
+        ],
+        "folder": "INBOX"
+    },
+    {
+        "numberInFolder": 6,
+        "title": "Re: TEST TEST TEST TEST TEST TEST TEST",
+        "content": [
+            "\nВАХ!!!!\n--\nОтправлено из Mail.Ru для Android пятница, 18 октября 2019г., 15:10 +03:00 от  ikpi63_holding@rambler.ru :\n\n>TeSt TeSt TeSt TeSt TeSt TeSt TeSt TeSt TeSt",
+            "\n<HTML><BODY><p style=\"margin-top: 0px;\" dir=\"ltr\">ВАХ!!!!</p> \n<div id=\"mail-app-auto-default-signature\">\n <p dir=\"ltr\">--<br> Отправлено из Mail.Ru для Android</p>\n</div>пятница, 18 октября 2019г., 15:10 +03:00 от <a href=\"mailto:ikpi63_holding@rambler.ru\">ikpi63_holding@rambler.ru</a>:<br><br><blockquote id=\"mail-app-auto-quote\" style=\"border-left:1px solid #0857A6; margin:0px 0px 0px 10px; padding:0px 0px 0px 10px;\" cite=\"15714006590000011926\">\n\n\n\n\n\n\n<div class=\"js-helper js-readmsg-msg\">\n\t<style type=\"text/css\"></style>\n \t<div >\n\t\t<base target=\"_self\" href=\"https://e.mail.ru/\" />\n\t\t\n            <div id=\"style_15714006590000011926_BODY\"><div class=\"class_1571424776\">\nTeSt TeSt TeSt TeSt TeSt TeSt TeSt TeSt TeSt\n</div></div>\n            \n        \n\t\t<base target=\"_self\" href=\"https://e.mail.ru/\" />\n\t</div>\n\n\t\n</div>\n\n\n</blockquote></BODY></HTML>\n"
+        ],
+        "contentType": "multipart/alternative; \r\n\tboundary=--ALT--A8a8ryORh94C2RIOb3FHcDKJCd8D88OC1571400838",
+        "date": "2019-10-18T12:13:58.000+0000",
+        "from": [
+            "Yuriy Borodyansky <borodyanskyum@gmail.com>"
+        ],
+        "recipients": [
+            "ikpi63_holding@rambler.ru"
+        ],
+        "folder": "INBOX"
+    }
+];
```

Comment:
Have u enter all this data manually? Where is it from?

Reply:
This data was given to me by Max, because there is no way to get them by the request.

Reply:
Ok thx